testreport/models.py

```diff
@@ -147,6 +147,20 @@ def __str__(self):
         return '{0} -> Launch: {1}'.format(self.test_plan, self.pk)
 
 
+class Build(models.Model):
+    launch = models.OneToOneField(Launch, related_name='build')
+    version = models.CharField(
+        max_length=128, default=None, null=True, blank=True)
```

Comment:
128 is too much for version, x - major, y - minor, z - minor-minor, so 123.123.123(10 - 16 symbols) is good enough.